### PR TITLE
Replace Web3Helper with module functions from utils

### PR DESCRIPTION
**Pull request by DMats**
Towards https://github.com/oceanprotocol/ocean.py/issues/304

### CODE REVIEW

**Comment by calina-c** [comment]
Can you please solve the conflicts and check whether Web3Helper is not used in other files in the `stable-aqua` branch?

Honestly, what I would do is redo the Web3Helper replacement work directly from stable aqua, to guarantee that there are no code conflicts and logic conflicts. Do you agree @DMats ?

**Comment by DMats** [comment]
Yeah that makes total sense. I agree.

File: aquarius/events/util.py [hunk 1]
```diff
@@ -10,12 +10,12 @@
 from ocean_lib.config import Config
 from ocean_lib.config_provider import ConfigProvider
 from ocean_lib.web3_internal.contract_handler import ContractHandler
+import ocean_lib.web3_internal.utils
```
Comment (calina-c) [inline aquarius/events/util.py:13]: why not use the same "from ... import ..." syntax?

Comment (DMats) [reply aquarius/events/util.py:13]: Name collision between `ocean_lib.web3_internal.utils.get_network_name` and `aquarius.events.util.get_network_name`

Comment (calina-c) [reply aquarius/events/util.py:13]: you can use "as" I think

**Review by calina-c — APPROVED** [review]

**Pull request merged**